app/components/add-house.js

```diff
@@ -36,3 +38,8 @@ export default class AddHouse extends React.Component {
     );
   }
 }
+
+export default connect(
+  undefined,
```

Comment:
you used `null` in the first file, but `undefined` here, you think this is a game?